src/content/en/scope/index.md

```diff
@@ -15,6 +15,9 @@ type: "section"
 |[testnetfaucet](https://github.com/decred/testnetfaucet)|<https://faucet.decred.org>|
 |[dcrdata](https://github.com/decred/dcrdata) *(latest stable release branch only)*|<https://explorer.dcrdata.org>|
 |[dcrtime](https://github.com/decred/dcrtime)|<https://time.decred.org>|
+|[dcrios *](https://github.com/planetdecred/dcrios) | <https://apps.apple.com/us/app/decred-wallet/id1462247643>|
+|[dcrandroid *](https://github.com/planetdecred/dcrandroid)| <https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet>|
+|[cspp](https://github.com/decred/cspp/)||
```

Comment:
Add link for cspp? https://cspp.decred.org/

Reply:
Id rather not. I will lead to people submitting issues like xframe on that site (which does not need) instead of finding vulnerabilities in the CSPP implementation.